homeassistant/components/ozw/lock.py

```diff
@@ -21,6 +36,23 @@ def async_add_lock(value):
         async_dispatcher_connect(hass, f"{DOMAIN}_new_{LOCK_DOMAIN}", async_add_lock)
     )
 
+    platform = entity_platform.current_platform.get()
+
+    platform.async_register_entity_service(
```

Comment:
The classic "after-merge" oh-moment. We need to add these services to the `ozw/services.yml` file.

Reply:
Oops, do I need a new PR for that or?

Reply:
Yes

Reply:
On it

Reply:
https://github.com/home-assistant/core/pull/37667